ui/analyse/css/study/relay/_embed.scss

```diff
@@ -58,7 +58,9 @@ main.is-relay.has-relay-tour {
     grid-template-areas: 'relay-tour . side';
 
     .relay-games {
-      max-height: unset;
+      @media (hover: hover) and (pointer: fine) {
```

Comment:
not sure if this unset is needed as I couldn't test it.  I didn't know how to set up a mock relay locally

Reply:
thanks @fitztrev this was really helpful!